### ISSUE TITLE
aria-label for date_select

### ISSUE THREAD

**Issue by dkniffin** [formtastic/formtastic#1350]
I might be missing something, but I believe there is currently no way to set `aria-label` for a `date_select` input. If I'm reading the code correctly, [this](https://github.com/formtastic/formtastic/blob/master/lib/formtastic/inputs/base/timeish.rb#L168-L171) is the method for determining the HTML options for those select inputs, and I don't see any way to change each input separately.

Assuming I'm correct on that, I'd like to request that feature be added, for accessibility reasons.

**Comment by justinfrench** [comment]
@dkniffin Sorry for the **long** delay replying.

Yep, very happy to add this, and even happy to help work up a PR.

I think it'll be difficult (or at least complex) to add-in support for each element separately, but I suspect we could just "do the right thing" and add the right attributes by default. Either way, are you able to help by describing in more detail either:

* The API you'd like to have for setting these, or
* The additional attributes you'd like to see in the default mark-up

Given how old this issue is I'm going to close for now, but will gladly re-open or start work on a PR if there's sufficient detail for me to get started.

**Comment by dkniffin** [comment]
Honestly, I don't even remember opening this 😅 I think it can stay closed until someone else requests it, unless you feel inclined to dig into it yourself.
